### PR TITLE
Make system text json version compatible with powershell

### PR DESCRIPTION
**Pull request by peterrsongg**
## Description

This adds an empty overrides.targets file so that in the build system we can override the published system.text.json dependency to 8.0.5. 
## Motivation and Context

Dotnet7830
## Testing

Tested that after E2E tests run the STJ version in the zip file is 8.0.5
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/aws/aws-sdk-net/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: sdk/src/Core/overrides.targets [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!--
+ this is a placeholder file to override the system text json version for netstandard20 and net472 targets in the build system.
```
[the remark this answers is not in the document]
Comment (peterrsongg) [reply sdk/src/Core/overrides.targets]: updated in latest commit

**Review by dscpinheiro — APPROVED** [review]

File: sdk/src/Core/AWSSDK.Core.NetFramework.csproj [hunk 2]
```diff
@@ -71,6 +71,7 @@
   <ItemGroup>
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Include="System.Memory" Version="4.5.5" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Text.Json" Version="6.0.11" />
```
Comment (normj) [inline sdk/src/Core/AWSSDK.Core.NetFramework.csproj:74]: Can you add a comment here explaining why we are locking to 6.0.11 which will be different then the nuspec file? This seems like something we will forget why in a few years from now.

File: sdk/src/Core/AWSSDK.Core.NetStandard.csproj [hunk 3]
```diff
@@ -83,6 +83,7 @@
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Include="System.Memory" Version="4.5.5" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Text.Json" Version="6.0.11" />
```
Comment (normj) [inline sdk/src/Core/AWSSDK.Core.NetStandard.csproj:86]: Can you add a comment here explaining why we are locking to 6.0.11 which will be different then the nuspec file? This seems like something we will forget why in a few years from now.

File: sdk/src/Core/overrides.targets [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<!--
+ this is a placeholder file to override the System.Text.Json version for netstandard20 and net472 targets in the build system.
+-->
+<Project>
```
Comment (normj) [inline sdk/src/Core/overrides.targets:4]: I'm unfamiliar on how having this empty file makes so the build system includes version 8.0.5. Can you give me the brief description on how this is used?

File: sdk/src/Core/AWSSDK.Core.NetFramework.csproj (hunk 2 above)
Comment (peterrsongg) [reply sdk/src/Core/AWSSDK.Core.NetFramework.csproj:78]: added comment

File: sdk/src/Core/overrides.targets (hunk 4 above)
Comment (peterrsongg) [reply sdk/src/Core/overrides.targets:5]: ^^ yup, i did add a bit more context though

File: sdk/src/Core/AWSSDK.Core.NetFramework.csproj (hunk 2 above)
Comment (afroz429) [reply sdk/src/Core/AWSSDK.Core.NetFramework.csproj:78]: Correction: Windows PowerShell doesn't ship with `System.Text.Json`, its the pwsh. It used to be called PowerShell Core but now its just called PowerShell. You could say `PowerShell(pwsh)`

Comment (peterrsongg) [reply sdk/src/Core/AWSSDK.Core.NetFramework.csproj:78]: ah okay, thanks for the correction. I will make an update!

**Review by afroz429 — APPROVED** [review]

**Pull request merged**